src/EventTickets/Repositories/EventTicketTypeRepository.php

```diff
@@ -26,8 +24,6 @@ class EventTicketTypeRepository
      */
     private $requiredProperties = [
         'event_id',
-        'created_at',
-        'updated_at',
```

Comment:
Why did you remove `created_at` and `updated_at` as required fields? In the database migration for the Event Ticket Type table these columns are defined as `NOT NULL`.

Reply:
The validation method that uses this array is called before the `createdAt` and `updatedAt` values are even generated. If we don't pass those values when creating a new `EventTicketType`, it will trigger errors because the required properties haven't been provided. However, the expected behavior is for the system to automatically generate those values for us.

Reply:
Hm. Good point. If the validation is running before it is generated that makes sense.